tests/unit/_base/all.js

```diff
@@ -4,6 +4,7 @@ define([
 	'./array',
 	'./connect',
 	'./declare',
+	'./declare-strict',
```

Comment:
Tests should run in all environments - when people write code using the new syntax that is safe for strict mode environments, we also want to make sure that code works in environments that don't support strict mode.

The only change is a few tests that check for errors being thrown - if strict mode is supported, incorrect syntax results in an error, and we test for that. If strict mode is not supported, no error is thrown (the code works without issue), so I changed the test expectations depending on whether or not the environment supports strict mode.